typings/balena-sdk/models.d.ts

```diff
@@ -307,6 +307,8 @@ declare module './index' {
 		device_tag: ReverseNavigationResource<DeviceTag>;
 		manages__device: ReverseNavigationResource<Device>;
 		service_install: ReverseNavigationResource<ServiceInstall>;
+		image_install: ReverseNavigationResource<ImageInstall>;
+		gateway_download: ReverseNavigationResource<GatewayDownload>;
```

Comment:
I think these should be
```suggestion
		image_install?: ReverseNavigationResource<ImageInstall>;
		gateway_download?: ReverseNavigationResource<GatewayDownload>;
```
to be compatible with the previous definition, and then in the places that explicit need these props you can use the `AtLeast` helper to type that they're required